src/app/mycart/my-cart.component.html

```diff
@@ -1 +1,24 @@
-<p class="mycart">mycart works!</p>
+
+<h1 class="mycart">MyCart</h1>
+
+<div class="table-body" *ngFor="let item of cartValues">
+    <div class="table-details">
+        <div><b>Product Name : </b>{{item.name}}</div>
+        <div class="quantity"><b>Quantity : </b>
+            <button (click)="decrement(item)">-</button>
+                {{item.quantity}}
+            <button (click)="increment(item)" >+</button>
+        </div>
+        <div><b>Price : </b>{{item.price}}</div>
+        <div><b>Total Price : </b>{{total(item.quantity,item.price)}}</div>
+    </div>  
+    <div class="action">
+        <button class="delete" (click)="productToDelete(item.id)">Delete</button>
+    </div>
+</div> 
+<div class="total">Total Amount : {{sum()}}</div>
```

Comment:
You shouldn't call functions from the template unless they are based on an event, like click, scroll etc. Template functions will get triggered on every change detection cycle. Use a class property instead.